test/classes/initializers/constAndParamInHeader.good

```diff
@@ -0,0 +1,4 @@
+constAndParamInHeader.chpl:6: In initializer 'Foo':
+constAndParamInHeader.chpl:6: error: invalid access of class member in initializer header
+constAndParamInHeader.chpl:14: In initializer 'Bar':
+constAndParamInHeader.chpl:14: error: invalid access of class member in initializer header
```

Comment:
It doesn't matter for now, but I'm not sure I'd use the term 'header' in this error message. Maybe "default argument" or "formal" or "function argument declaration".